pkg/graph/schema.resolvers.go

```diff
@@ -1585,12 +1589,41 @@ func (r *queryResolver) DetailedCedarSystemInfo(ctx context.Context, cedarSystem
 	return &dCedarSys, nil
 }
 
-func (r *queryResolver) SystemIntakeContacts(ctx context.Context, id uuid.UUID) ([]*models.SystemIntakeContact, error) {
+func (r *queryResolver) SystemIntakeContacts(ctx context.Context, id uuid.UUID) ([]*models.AugmentedSystemIntakeContact, error) {
 	contacts, err := r.store.FetchSystemIntakeContactsBySystemIntakeID(ctx, id)
 	if err != nil {
 		return nil, err
 	}
-	return contacts, nil
+
+	euaIDs := make([]string, len(contacts))
+	for i, contact := range contacts {
+		euaIDs[i] = contact.EUAUserID
+	}
+
+	userInfos, err := r.service.FetchUserInfos(ctx, euaIDs)
+	if err != nil {
+		return nil, err
+	}
+
+	userInfoMap := make(map[string]*models.UserInfo)
+	for _, userInfo := range userInfos {
+		if userInfo != nil {
+			userInfoMap[userInfo.EuaUserID] = userInfo
+		}
+	}
+
+	augmentedContacts := make([]*models.AugmentedSystemIntakeContact, len(contacts))
```

Comment:
There's a subtle bug here; if any of the listed contacts are no longer valid in CEDAR LDAP, this array will end up having some `nil` entries, because `userInfoMap` won't have any entries for those contacts' EUA IDs. To verify:
* Use the real CEDAR client instead of the local fake. Easiest way to do this is commenting out the setup code in `pkg/server/routes.go` that calls `local.NewCedarLdapClient`.
* Insert two contacts, one with a valid EUA ID, one without, by running the GQL mutation in this PR's description.
* Run the GQL query to fetch contacts from this PR's description. You should get this error result:
```
{
  "errors": [
    {
      "message": "must not be null",
      "path": [
        "systemIntakeContacts",
        0
      ]
    }
  ],
  "data": null
}
```

I'm not sure what the best way to fix this is. It's pretty easy to filter out the invalid EUA IDs (initialize `augmentedContacts` as an empty array and append to it, instead of setting the `i`'th element), but we probably want to indicate in the GQL response which EUA IDs (if any) are invalid. Maybe we have a type in our GraphQL schema that's like:

```
type SystemIntakeContacts {
  validContacts: [AugmentedSystemIntakeContact!]!
  invalidEUAIDs: [String!]!
}
```
and return that from the `systemIntakeContacts` query?

Reply:
(Incidentally, don't feel bad about this bug, I pretty much only noticed it by accident)